spec/models/item_spec.rb

```diff
@@ -4,7 +4,8 @@
   subject do
     described_class.new(name: "Anything",
```

Comment:
This sort of style can be hard to maintain moving forward. If the name of the variable / method you are calling ever changes, you will need to realign you indentation. As a general rule of thumb, I would try to be consistent with your indentation, and in the case of this project, always indent using two spaces.

Here's what I would do instead:
```ruby
described_class.new(
  name: "Anything",
  sell_in: 10,
  quality: 50,
  description: "Something Else"
)
```